### PR TITLE
Adding Request headers

### PR DESCRIPTION
**Pull request by n04ln**
This PR for Today's SCCP classroom.

First, I'm sorry, I cannot attend this class. :bow: 

Therefore, I commented this PR.

/cc @mt-coff

### CODE REVIEW

File: request-headers/app.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+
+@app.route("/content-type", methods=["POST"])
+def content_type():
+    # Content-Typeがapplication/jsonでないならerrorを返す
+    if not request.headers.get("Content-Type") == 'application/json':
+        # JSON形式でエラーを返す
+        return make_response(jsonify(res="error"), 400)
+    # リクエストから飛んできたContent-Typeを返す
+    return request.headers["Content-Type"]
```
Comment (n04ln) [inline request-headers/app.py:18]: if u have a lot of time. I recommend practice using other headers. (e.g, `Authorization`)

and if u explain about `Authorization` header, plz talk about value format. (e.g, `Authorization: Bearer XXXX`) 
ref. https://openid-foundation-japan.github.io/rfc6750.ja.html (japanese)

Comment (n04ln) [reply request-headers/app.py:18]: /cc @mt-coff

File: request-headers/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+### 資料リンク
```
Comment (n04ln) [inline request-headers/README.md:1]: What's this? 
/cc @mt-coff

Comment (mt-coff) [reply request-headers/README.md:1]: I will add database document.
comming soon...

Comment (n04ln) [reply request-headers/README.md:1]: OK :)

Thanks for adding documents 👍

**Comment by n04ln** [comment]
if u added documents for today's class, I review it.
so, plz mention to me!

/cc @mt-coff

**Comment by mt-coff** [comment]
I added docs :). plz check changes!! @NoahOrberg @NoahOrberg @NoahOrberg :fire:

**Comment by n04ln** [comment]
LGTM

**Pull request merged**